client/my-sites/plugins/plugin-site-disabled-manage/index.jsx

```diff
@@ -44,4 +46,8 @@ const PluginSiteDisabledManage = ( { isNetwork, site, plugin, translate } ) => {
 	);
 };
 
-export default localize( PluginSiteDisabledManage );
+export default connect(
+	( state, ownProps ) => ( {
+		remoteManagementUrl: getJetpackSiteRemoteManagementUrl( state, ownProps.site.ID )
```

Comment:
Usually it's good to check for `site` before using it (because it might be `null` if it's not loaded yet. However, the components that render `PluginSiteDisabledManage` guard against this, so it will not be necessary in this case.